src/init/starship.zsh

```diff
@@ -24,7 +24,7 @@ else
 fi
 
 # Will be run before every prompt draw
```

Comment:
Can we modify this comment (or add one above it) to explain why the naming scheme is needed? It departs from the naming schemes we use in most of the other init scripts, so I'm worried someone will try to change it back to preserve styling.

Reply:
This OK? https://github.com/starship/starship/pull/3480/files#diff-b3ecc1f5c604bb3d4c571fac2010993e8d3858051f5f64e9f51ad8d5989e47f8R27